### ISSUE TITLE
Adding paddingBottom to page can cause rendering issues

### ISSUE THREAD

**Issue by dberringer** [diegomura/react-pdf#739]
**Describe the bug**
I have noticed that if I add paddingBotom to the page component the page wrapping can be wonky depending on how things land at the end of the page.  For example a borderBottom will render at the bottom of the previous page as well as at the top of the next page.  See attached screenshot.

**To Reproduce**
If you remove the paddingBottom from the page style in my example it will render normally.

_You can make use of [react-pdf REPL](https://react-pdf/repl) to share the snippet_
https://react-pdf.org/repl?code=3187b0760ce02e00402a05368c0bc30050129503e0c0a061801e0044460057016c130a1d0a24801404301cc119a013c01b04280379f411001d00074e0802fa366cd84041004e6adaf096ca54febdd184afdf801a18eb361806c001932609d577a0c012ca026a17dd8002608001ed82878c24c4a4aa090208212fc201ce89edebe01c19800dc9151446a0850946a6018c48a7951c488410c88d07e1cc27e814172121dc400f4350cb99524bd3c500242c2c20046206a816a004220505020d4005c3000e4008c3c71eefe3013fc6cc000d61b720af5508dcd196d1d12ddbd1503dd38d9fdcc729872b9ddec2e055bae42a2d1e88c4fbe00062e0280480a1c77340106a7404488003336351dc067586c00f210003b9b1f8fe0d99898103530009000b2594820ab2e9753170c9071a06c1bb0024a06a174205d10293c9fe2e800dcb600662e8013400fa700034800d46c200007001d440d28980155fc72ea32a001a004e360a8541225a6236f81f8e5f0316418890a8180019446823f432108541414f90874530645c7586398327f3f91a7010149d600163b352944994d80380b258acb339dcbe71a001904262a0eb002b396f36c64e34004aee0e133d672e6cc0e4d4b77e1f0ed8409ca02c52342117459ba1aa48581743e3920000

**Expected behavior**
I would expect that the items that the borderBottom of the component that was pushed to the next page not display on the previous page too.

**Screenshots**
![Screen Shot 2019-10-16 at 3 48 33 PM](https://user-images.githubusercontent.com/108108/66965208-b86f3080-f02d-11e9-8d9c-5c3251f04c20.png)
![Screen Shot 2019-10-16 at 3 54 21 PM](https://user-images.githubusercontent.com/108108/66965213-ba38f400-f02d-11e9-89a1-d13018c53b7b.png)

**Desktop (please complete the following information):**
 - OS: [MacOS]
 - Browser [e.g. firefox]
 - React-pdf version [e.g. v1.6.4]

**Comment by dulcineapena1** [comment]
@dberringer 
I solved that using `wrap` prop in View component. This will transform breakable elements into unbrakable.
```
<View wrap={false}>
      <Text>Testing{index}...</Text>
      <Text style={{borderBottom: '1 solid black'}}>Testing{index}...</Text>
</View>
```
Reference: https://react-pdf.org/advanced#page-wrapping

**Comment by dberringer** [comment]
Hey @dulcineapena1 thanks for the suggestion but it actually misses the issue.  I'm aware I can use wrap to prevent the two lines from breaking but the point is that if they are supposed to break and do break they don't display correctly.  Is anyone else noticing this?

**Comment by dberringer** [comment]
Hey @diegomura.  Just checking back to see if you have any thoughts on how/when we might get a fix for this?

**Comment by KanishkVashisht** [comment]
Facing a similar issue.

**Comment by mstrk** [comment]
I'm also fighting to make views respect the page padding bottom when wrapping.

**Comment by jeremy-christian** [comment]
Having a similar issue here, adding bottom padding / margin to my pages causes them not to wrap at all...

**Comment by diegomura** [comment]
Fixed in 2.0! Please check your [REPL example](https://react-pdf.org/repl?code=3187b0760ce02e00402a05368c0bc30050129503e0c0a061801e0044460057016c130a1d0a24801404301cc119a013c01b04280379f411001d00074e0802fa366cd84041004e6adaf096ca54febdd184afdf801a18eb361806c001932609d577a0c012ca026a17dd8002608001ed82878c24c4a4aa090208212fc201ce89edebe01c19800dc9151446a0850946a6018c48a7951c488410c88d07e1cc27e814172121dc400f4350cb99524bd3c500242c2c20046206a816a004220505020d4005c3000e4008c3c71eefe3013fc6cc000d61b720af5508dcd196d1d12ddbd1503dd38d9fdcc729872b9ddec2e055bae42a2d1e88c4fbe00062e0280480a1c77340106a7404488003336351dc067586c00f210003b9b1f8fe0d99898103530009000b2594820ab2e9753170c9071a06c1bb0024a06a174205d10293c9fe2e800dcb600662e8013400fa700034800d46c200007001d440d28980155fc72ea32a001a004e360a8541225a6236f81f8e5f0316418890a8180019446823f432108541414f90874530645c7586398327f3f91a7010149d600163b352944994d80380b258acb339dcbe71a001904262a0eb002b396f36c64e34004aee0e133d672e6cc0e4d4b77e1f0ed8409ca02c52342117459ba1aa48581743e3920000)
